src/Host.cpp

```diff
@@ -471,6 +471,34 @@ void Host::waitForProfileSave()
     }
 }
 
+void Host::setMmpMapLocation(const QString &var)
+{
+    auto document = QJsonDocument::fromJson(var.toUtf8());
+    if (!document.isObject()) {
+        return;
+    }
+    auto json = document.object();
+    if (json.isEmpty()) {
+        return;
+    }
+
+    auto urlValue = json.value(QStringLiteral("url"));
+    if (urlValue == QJsonValue::Undefined) {
+        return;
+    }
+    auto url = QUrl(urlValue.toString());
+    if (!url.isValid()) {
+        return;
+    }
+
+    mpMap->setMmpMapLocation(urlValue.toString());
+}
+
+QString Host::mmpMapLocation() const
```

Comment:
Maybe `getMmpMapLocation` to keep consistent with the setter?

Reply:
Qt style doesn't seem to use a get, it just uses it as-is, and that seems okay to me

Reply:
At least at lua side we use get. And I thought in C++ we did as well until now.

Reply:
Hm, we do use `get`. I'll change it for consistency.